app/assets/stylesheets/common/_login.scss

```diff
@@ -3,8 +3,8 @@
  */
 
 .login {
-  @include border-radius($radius);
-  @include box-shadow(0 3px 0 0 $grey);
+  border-radius: $radius;
```

Comment:
Properties should be ordered background, border-radius, box-shadow, height, left, margin-left, margin-top, position, top, width